apps/site/assets/ts/stop/components/amenities/AmenityCard.tsx

```diff
@@ -22,15 +53,25 @@ const AmenityCard = ({
             {headerText}
           </div>
         </div>
-        {children && (
-          <div className="c-descriptive-link__text hidden-sm-down">
-            {children}
+        {content && (
+          <div className="c-descriptive-link__text pt-8 hidden-sm-down">
+            {content}
           </div>
         )}
       </div>
-      <div className="c-descriptive-link__caret-wrapper">
+      <button
+        type="button"
+        className="c-descriptive-link__caret-wrapper"
+        onClick={() => setModalOpen(true)}
+        disabled={!children}
+      >
```

Comment:
question: do we want just the button clickable or the whole card?

Reply:
just reviewed the docs and it looks like we want the whole card to be clickable, good call. will revise